codecparsers/h265Parser.cpp

```diff
@@ -609,12 +631,15 @@ bool Parser::scalingListData(ScalingList* dest_scaling_list, NalReader& nr)
                 coefNum = std::min(64, (1 << (4 + (sizeId << 1))));
 
                 if (sizeId > 1) {
-                    dstDcList[matrixId] = nr.readSe();
-                    nextCoef = dstDcList[matrixId] + 8;
+                    int32_t tmp;
```

Comment:
could we rename tmp to scaling_list_dc_coef_minus8  and make assign dstDcList like follow:
dstDcList[matrixId] = scaling_list_dc_coef_minus8 + 8;